src/main/java/uk/gov/cshr/civilservant/validation/ValidAgencyTokenCapacityValidator.java

```diff
@@ -17,11 +18,14 @@
 
     @Override
     public void initialize(ValidCapacity constraintAnnotation) {
+
     }
 
     @Override
     public boolean isValid(Integer value, ConstraintValidatorContext context) {
+        ((ConstraintValidatorContextImpl) context).addMessageParameter("minValue", minValue);
```

Comment:
Depending on how expensive this call is, could be optimised by only adding these values in the event the passed value is not valid.

Reply:
Agreed, optimisation updated.